pkg/codegen/go/gen.go

```diff
@@ -2912,15 +2906,20 @@ func (pkg *pkgContext) getImports(member interface{}, importsAndAliases map[stri
 	}
 }
 
-func (pkg *pkgContext) genHeader(w io.Writer, goImports []string, importsAndAliases map[string]string) {
+func (pkg *pkgContext) genHeader(w io.Writer, goImports []string, importsAndAliases map[string]string, isUtil ...bool) {
```

Comment:
```suggestion
func (pkg *pkgContext) genHeader(w io.Writer, goImports []string, importsAndAliases map[string]string, isUtil bool) {
```

Let's just update ever call site. 
```go
notUtil := false
genHeader(w, goImports, importsAndAliases, notUtil)
```

That this is treated as `notUtil = true` is super confusing. I have no trouble believing that someone will want to disable `isUtil` and change a true to a false.

Reply:
happy to be explicit here! 😂